src/server/api/routers/auth.ts

```
@@ -234,13 +235,15 @@ export const authRouter = createTRPCRouter({
       if (!isDev) {
         const { sendEmail } = await import("../../_lib/email");
         const { renderMobileOtpEmail } = await import("../../_lib/emails");
+        logger.info("Sending OTP email", { email: normalizedEmail });
         const html = await renderMobileOtpEmail({ code });
         const emailResult = await sendEmail({ email: normalizedEmail, subject: `${code} is your ISO sign-in code`, html });
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'html' should be before 'subject'.

(sort-keys-fix/sort-keys-fix)